src/components/buttons/ProfileButton/ProfilePopup.tsx

```diff
@@ -0,0 +1,84 @@
+import styled from '@emotion/styled';
+
+import { logoutAPI } from '@/apis/user';
+import useUserStore from '@/stores/user';
+import { FONT_REGULAR_3, FONT_REGULAR_4, FONT_REGULAR_5 } from '@/styles/font';
+
+const ProfilePopup = () => {
+  const { user, reset } = useUserStore();
+  const onLogout = async () => {
+    try {
+      await logoutAPI();
+      reset();
+    } catch (e) {
+      alert('로그아웃에 실패했습니다');
+    }
+  };
```

Comment:
로그아웃 기능은 제가 `hooks/useLogout.ts`에 미리 작성해두었습니다!

아래와 같은 방법을 사용하시는 것은 어떤가요?
```suggestion
  const onLogout = useLogout();
```

대신 로그아웃 실패 작업이 요구될 경우 hook 변경 or 추가 함수 작성이 필요할 수 있습니다.

Reply:
말씀하신 방법으로 수정하겠습니다!